code-challenges.js

```diff
@@ -17,7 +17,24 @@ const fruit3 = "cherry"
 const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
-// Explain your code:
+
+//input: 2 strings
+//output: string that has the most characters
+//create a function that takes in both strings
+//conditional statement to tell me which string has more characters using .length method and > operator 
+// return using string interpolation
+
+const fruitEvaluator = (string1, string2,) => {
+    if (string1.length > string2.length){
+        return `${string1} has more characters than ${string2}` 
+    } else if (string2.length > string1.length) {
+        return `${string2} has more characters than ${string1}`
+    } else ("entry is unknown")
+}
+    console.log(fruitEvaluator(fruit1, fruit2)) //output: banana has more characters than apple
+    console.log(fruitEvaluator(fruit3, fruit4)) //output: cherry has more characters than kiwi
+
+// Explain your code: The question states to compare two strings and evaluate which one has more characters. I did this within my function using a conditional statment. it tells the computer to return which string has more characters by using the > operator and string interpolation within the return. 
```

Comment:
great description, Ryan!  Your code looks great.  Push yourself to cover edge cases (i.e. 2 equal numbers). Well done!